src/core/server/path.ts

```diff
@@ -18,34 +18,53 @@
  */
 
 import { join } from 'path';
-import { accessSync, R_OK } from 'fs';
-import { find } from 'lodash';
-import { fromRoot } from '../../utils';
+import { accessSync, constants } from 'fs';
+import { TypeOf, schema } from '@kbn/config-schema';
+import { fromRoot } from '../utils';
```

Comment:
Not related to this PR: Interesting that it's not in the `server/utils` folder. As well as `package_json`. I'd expect only env agnostic code in `src/core/utils`. There is `src/core/public/utils` for browser-specific utils. We don't have `src/core/server/utils` as of now. Probably `fromRoot` and `pkg` should move there.

Reply:
Should we create an issue to handle this?

Reply:
let's do in this PR. because you just introduced them in this PR